### PR TITLE
MEPTS 1218 | MQ Cat 13 Diff ICAP DB

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java [hunk 1]
```diff
@@ -2326,6 +2326,38 @@ public static String getFirstLineTherapyQuery() {
...
+        + "   AND gest.first_gestante <= :endDate "
+        + "   AND gest.first_gestante > DATE_ADD((SELECT MIN(o.value_datetime) as art_date "
+        + "                                       FROM encounter e "
+        + "                                           INNER JOIN obs o ON e.encounter_id = o.encounter_id "
+        + "                                       WHERE gest.patient_id = e.patient_id "
+        + "                                         AND e.voided = 0 AND o.voided = 0 "
+        + "                                         AND e.encounter_type = ${53} AND o.concept_id = ${1190} "
+        + "                                         AND o.value_datetime IS NOT NULL AND o.value_datetime <= :endDate AND e.location_id = :location "
+        + "                                       LIMIT 1), interval 3 MONTH) ";
+  }
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: Please put the spec of this method

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
@TinoMuianga please put the specs of the pregnancy method

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Looks good to me, Please put the specs

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java (hunk 1 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/QualityImprovement2020Queries.java]: Done

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**